go.mod

```diff
@@ -19,3 +19,5 @@
 module github.com/apache/plc4x
 
 go 1.15
+
```

Comment:
This toplevel go.mod file confuses me. What is the purpose of that? I open plc4go as TLP in golang so maybe this is from opening the whole plc4x in goland?